app/classifier/tasks/survey/editor.cjsx

```diff
@@ -14,97 +14,184 @@ Papa = require 'papaparse'
+    characteristicsName: null
+    confusionsName: null
+    questionsName: null
+
+  componentWillReceiveProps: (nextProps) ->
+    if nextProps.task isnt @props.task
+      @clearState()
+      @setState
+        task: JSON.parse JSON.stringify nextProps.task
+
   render: ->
-    window.editingSurveyTask = @props.task
+    contentWarnings = @checkContentWarnings()
 
     <div className="workflow-task-editor">
-      <p><span className="form-label">Import task data. See <a href="https://www.zooniverse.org/projects/aliburchard/cameratraptest/faq">here for help</a>.</span></p>
-      <div className="columns-container" style={marginBottom: '0.2em'}>
-        <FileButton className="major-button column" accept=".csv, .tsv" multiple onSelect={@handleFiles.bind this, @addChoice, null}>Add choices CSV</FileButton>
-        <TriggeredModalForm trigger={
-          <span className="secret-button">
-            <i className="fa fa-question-circle"></i>
-          </span>
-        }>
-          {surveyEditorHelp.choices}
-        </TriggeredModalForm>
-      </div>
-      <div className="columns-container" style={marginBottom: '0.2em'}>
-        <FileButton className="major-button column" accept=".csv, .tsv" multiple onSelect={@handleFiles.bind this, @addCharacteristics, null}>Add characteristics CSV</FileButton>
-        <TriggeredModalForm trigger={
-          <span className="secret-button">
-            <i className="fa fa-question-circle"></i>
-          </span>
-        }>
-          {surveyEditorHelp.characteristics}
-        </TriggeredModalForm>
-      </div>
-      <div className="columns-container" style={marginBottom: '0.2em'}>
-        <FileButton className="major-button column" accept=".csv, .tsv" multiple onSelect={@handleFiles.bind this, @addConfusion, null}>Add confused pairs CSV</FileButton>
-        <TriggeredModalForm trigger={
-          <span className="secret-button">
-            <i className="fa fa-question-circle"></i>
-          </span>
-        }>
-          {surveyEditorHelp.confusions}
-        </TriggeredModalForm>
-      </div>
-      <div className="columns-container" style={marginBottom: '0.2em'}>
-        <FileButton className="major-button column" multiple onSelect={@handleFiles.bind this, @addQuestion, @cleanQuestions}>Add questions CSV</FileButton>
-        <TriggeredModalForm trigger={
-          <span className="secret-button">
-            <i className="fa fa-question-circle"></i>
-          </span>
-        }>
-          {surveyEditorHelp.questions}
-        </TriggeredModalForm>
-      </div>
-
+      <span className="form-label">INSTRUCTIONS</span>
+      <p><small>The survey task is ideal for asking volunteers to identify wildlife species in camera trap photos, but can be used anytime you have a lot of choices you need volunteers to filter from.</small></p>
+      <p><small><a href="https://www.zooniverse.org/projects/aliburchard/cameratraptest/about/faq">See here</a> for detailed instructions.</small></p>
```

Comment:
we probably shouldn't link to documentation inside a particular project--especially a test project!--in the general project editor UI, is there somewhere else we can host this content? (not necessary before merge)